tools/docker/builder/openwrt/Dockerfile

```diff
@@ -69,47 +69,42 @@ RUN git clone "$OPENWRT_REPOSITORY" openwrt \
-    && echo "CONFIG_TARGET_${TARGET_SYSTEM}_${SUBTARGET}=y" >> .config \
-    && echo "CONFIG_TARGET_${TARGET_SYSTEM}_${SUBTARGET}_${TARGET_PROFILE}=y" >> .config \
-    && echo "CONFIG_PACKAGE_prplmesh${PRPLMESH_VARIANT}=y" >> .config \
-    && if [ "$TARGET_PROFILE" = "DEVICE_NETGEAR_RAX40" ] ; then \
-        grep -v -E 'CONFIG_SDK|CONFIG_MAKE_TOOLCHAIN' feeds/intel/rax40.profile >> .config ;\
-    fi ;\
-    : "Installing intel feed doesn't correctly regenerate kernel .package-info"; \
-    : "force regeneration by removing it"; \
-    rm -rf tmp \
-    && make defconfig; \
-    : "We need to keep the hashes in the firmware, to later know whether an upgrade is needed:"; \
-    mkdir -p files/etc \
+RUN mkdir -p files/etc \
+#   We need to keep the hashes in the firmware, to later know if an upgrade is needed:
     && printf '%s=%s\n' "OPENWRT_REPOSITORY" "$OPENWRT_REPOSITORY" >> files/etc/prplwrt-version \
     && printf '%s=%s\n' "OPENWRT_VERSION" "$OPENWRT_VERSION" >> files/etc/prplwrt-version \
-    && printf '%s=%s\n' "PRPLMESH_VARIANT" "$PRPLMESH_VARIANT" >> files/etc/prplwrt-version \
-    && printf '%s=%s\n' "PRPL_FEED" "$PRPL_FEED" >> files/etc/prplwrt-version \
-    && printf '%s=%s\n' "INTEL_FEED" "$INTEL_FEED" >> files/etc/prplwrt-version \
-    && printf '%s=%s\n' "IWLWAV_FEED" "$IWLWAV_FEED" >> files/etc/prplwrt-version \
-    && make -j$(nproc) \
-    && make package/feeds/prpl/prplmesh/clean
+    && if [ "$TARGET_PROFILE" = DEVICE_NETGEAR_RAX40 ] ; then \
+        sed -i 's/packages:/packages:\n  - prplmesh-dwpal/g' profiles/intel_mips.yml \
+        && yq write --inplace profiles/intel_mips.yml feeds -f profiles_feeds/netgear-rax40.yml \
+        && ./scripts/gen_config.py intel_mips \
+#       Installing intel feed doesn't correctly regenerate kernel .package-info
+#       force regeneration by removing it
+        && rm -rf tmp \
+#       For some reason we have to run gen_config a second time to get a correct .config:
+        && ./scripts/gen_config.py intel_mips \
+#       make sure intel's bridge-utils is the only one that gets installed:
+        && rm -rf ./package/feeds/packages/bridge-utils \
+        && scripts/feeds install -p feed_bridge_utils bridge-utils \
+        && cat profiles_feeds/netgear-rax40.yml >> files/etc/prplwrt-version ;\
+    else \
+        cp feeds.conf.default feeds.conf \
+        && echo "src-git prpl $PRPL_FEED" >> feeds.conf \
+        && scripts/feeds update -a \
+        && scripts/feeds install -a \
+        && echo "CONFIG_TARGET_${TARGET_SYSTEM}=y" >> .config \
+        && echo "CONFIG_TARGET_${TARGET_SYSTEM}_${SUBTARGET}=y" >> .config \
+        && echo "CONFIG_TARGET_${TARGET_SYSTEM}_${SUBTARGET}_${TARGET_PROFILE}=y" >> .config \
+        && echo "CONFIG_PACKAGE_prplmesh${PRPLMESH_VARIANT}=y" >> .config \
+        && make defconfig \
+        && printf '%s=%s\n' "PRPL_FEED" "$PRPL_FEED" >> files/etc/prplwrt-version ;\
+    fi ;\
+    make -j$(nproc) \
```

Comment:
Not really for this commit, maybe, but I think it would be useful to do this in a separate RUN. It makes the output cleaner.